packages/ui/feature-dashboard/src/lib/pages/flows-table/flows-table.component.ts

```diff
@@ -109,12 +111,14 @@ export class FlowsTableComponent implements OnInit {
 
   openBuilder(flow: Flow, event: MouseEvent) {
     const link = '/flows/' + flow.id;
-    if (event.ctrlKey || event.which == 2 || event.button == 4) {
+    if (
```

Comment:
Move this routing logic to separate service, along side is Embedded, as this function will be used multiple times.